lte/gateway/c/core/oai/tasks/sgw/sgw_task.c

```diff
@@ -271,3 +279,45 @@ static void spgw_app_exit(void) {
   OAI_FPRINTF_INFO("TASK_SPGW_APP terminated\n");
   pthread_exit(NULL);
 }
+
+static void start_state_hashtable_timer(void) {
+  state_hashtable_timer_id = start_timer(
+      &spgw_app_task_zmq_ctx, 1000 * state_hashtable_timer_sec,
+      TIMER_REPEAT_FOREVER, handle_state_hashtable_timer, NULL);
+}
+
+static int handle_state_hashtable_timer(zloop_t* loop, int id, void* arg) {
+  hash_table_ts_t* spgw_state_ue_ht = get_spgw_ue_state();
+  hashtable_key_array_t* keys       = hashtable_ts_get_keys(spgw_state_ue_ht);
+
+  hashtable_rc_t ht_rc;
+  uint64_t ue_id;
+  s_plus_p_gw_eps_bearer_context_information_t* ue_desc_p;
```

Comment:
I think we would simply want to monitor the number of elements stored in the hash table. Not sure what would be the utility of collecting all the keys and checking its data.